_posts/2019-04-16-preservation.md

```diff
@@ -0,0 +1,360 @@
+---
+date: 2019-04-16 8:56:00 -0400
+title: Preservation
+layout: default
+---
+
+## Preservation
+**by Trey Pendragon**
+
+Lately we've been working on preservation within
+[Figgy](https://github.com/pulibrary/figgy) and have come up with a strategy we
+think works for us. However, in doing so, we've evaluated a few options and come
+up with some assumptions which could be useful to others.
+
+<!--more-->
+
+### Object Boundaries
+
+I talk about an "object" a lot below. After much discussion, we decided at
+  Princeton that we want a "directory" to consist of a top-level object (like a
+  Book) and all its children.
+
+This means a requirement is that I'm able to copy a single directory and get
+everything I need to re-ingest that object and display it. External references
+may not come in, but the hierarchy will.
+
+So a Volume will be nested underneath its Multi-Volume Work, and a Page will be
+nested underneath its Book. This is important, because if one decides against
+this then the pros/cons of the following cases may change significantly.
+
+### Assumptions
```

Comment:
I think we should add something to this section about being able to verify that binary files are intact using their stored checksums.

Reply:
Added a small sentence - look good?